connman/vpn/plugins/openvpn.c

```diff
@@ -450,6 +450,15 @@ static int run_connect(struct ov_private_data *data,
 	const char *option;
 	int err = 0;
 
+	/*
+	 * Unless set explicitly by the user default to 10 auth errors after a
+	 * succesful connection attempt was made before clearing them via VPN
+	 * agent.
+	 */
+	option = vpn_provider_get_string(provider, "AuthErrorLimit");
+	if (!option)
+		vpn_provider_set_string(provider, "AuthErrorLimit", "10");
```

Comment:
I haven't looked if there's an easy way to do this in any other way, but done this way, the default value ends up being saved in the configuration file, and if we change the default, it won't change for existing services even if the value has not been explicitly set.

Reply:
There is, in 3c87042b46fd46316de0631d898c0b0c57bf8c8e I added `vpn_provider_set_auth_error_limit()` that is for plugins to set their defaults which is used only when "AuthErrorLimit" is not set by user and defaults to `AUTH_ERROR_LIMIT_DEFAULT` (renamed this as well) that is now 1. Added use of this to OpenVPN as well and to my real surprise, that AUTH_FAILED does occur also with ProtonVPN over TCP! Odd. I was under the assumption that this happens only with UDP hence the protocol design but it could be easily detected when TCP is used. Oh well. Maybe that could be documented in some comment,